OCPBUGS-2936: VPA uninstall instructions incomplete/complicated (#53447)
openshift/openshift-docs: modules/nodes-pods-vertical-autoscaler-uninstall.adoc

. Switch to the *openshift-vertical-pod-autoscaler* project.

> . Find the *VerticalPodAutoscaler*  Operator and click the Options menu {kebab} and select *Uninstall Operator*.

Author: @skrthomas It seems, based on Atom searches, that when referring to the kebab menu, the action (select, click, etc) is in the same sentence.

> . Find the *VerticalPodAutoscaler*  Operator and click the Options menu {kebab} and select *Uninstall Operator*.

Author: I think this sounds better:

> For the *VerticalPodAutoscaler*  Operator, click the Options menu {kebab} and select *Uninstall Operator*.

. Optional: To remove all operands associated with the Operator, in the dialog box, select *Delete all operand instances for this operator* checkbox.

. Click *Uninstall*.

. Optional: Use the OpenShift CLI to remove the VPA components:

.. Delete the VPA namespace: 
+
[source,terminal]
----
$ oc delete namespace openshift-vertical-pod-autoscaler
----

.. Delete the VPA custom resource definition (CRD) objects:
+
[source,terminal]
----
$ oc delete crd verticalpodautoscalercheckpoints.autoscaling.k8s.io
----
+
[source,terminal]
----
$ oc delete crd verticalpodautoscalercontrollers.autoscaling.openshift.io
----
+
[source,terminal]
----
$ oc delete crd verticalpodautoscalers.autoscaling.k8s.io
----
+
Deleting the CRDs removes the associated roles, cluster roles, and role bindings. 
+
[NOTE]
====

> This action removes from the cluster all user-created VPA CRs. If you re-install the VPA, you would need to create these objects again.

Reviewer: ```suggestion
This action removes from the cluster all user-created VPA CRs. If you re-install the VPA, you must create these objects again.
```
